tests/unit/test_charm.py

```diff
@@ -437,7 +437,7 @@ def test_get_resource_token_action__success(self):
         self.harness.enable_hooks()
 
         self.start_container()
-
+        del self.harness.charm._state.resource_token
```

Comment:
Why do we need this now?

Reply:
Because the `self.start_container` method sets it to a non-empty value. This line ensures we're doing things on a clean state.